app/client/src/pages/Editor/gitSync/GitSyncModal.tsx

```diff
@@ -33,10 +33,12 @@ const Container = styled.div`
 
 const BodyContainer = styled.div`
   flex: 3;
-  height: 100%;
+  height: calc(100% - 47px);
 `;
 
-const MenuContainer = styled.div``;
+const MenuContainer = styled.div`
+  height: 47px;
+`;
```

Comment:
nitpick: It might make if more readable if we assign a variable to offset values:
```suggestion
const MENU_HEIGHT = "47px";
const MenuContainer = styled.div`
  height: ${MENU_HEIGHT};
`;
```